src/test/java/persistence/study/ReflectionTest.java

```diff
@@ -0,0 +1,100 @@
+package persistence.study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ReflectionTest {
+    private static final Logger logger = LoggerFactory.getLogger(ReflectionTest.class);
+
+    @Test
+    @DisplayName("요구사항1. Car 객체 정보 가져오기")
+    void showClass() {
+        Class<Car> carClass = Car.class;
+        logger.debug(carClass.getName());
+        logger.debug(Arrays.toString(carClass.getDeclaredFields()));
+        logger.debug(Arrays.toString(carClass.getDeclaredConstructors()));
+        logger.debug(Arrays.toString(carClass.getDeclaredMethods()));
+    }
+
+    @Test
+    @DisplayName("요구사항2. test로 시작하는 메소드 실행")
+    void testMethodRun() throws NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
```

Comment:
```suggestion
    void testMethodRun() throws Exception {
```
사소하지만 저는 테스트에서는 위와 같이 최상위 예외로 throws 를 선언하는게 보기 좋더라구요!
개인적인 의견이니 참고만 부탁드려요 :)